### DISCUSSION TITLE
rsyncを使ってリモートからローカルでバックアップを取る

### DISCUSSION THREAD

**Discussion by TomokiMotegi** [nig-sc/nigsc_homepage2#478] [General]
いつもお世話になっております。
定期的に自身のローカルにバックアップを取る体制にしたいのですが、推奨されている方法はありますでしょうか？
表題の方法を考えているのですが、そちらへの負担が小さい方法をご教示いただければ幸いです。

茂木

**Comment by sc-fs** [comment]
以下にデータ転送に関する記載がありますので、参考にしてください。
https://sc.ddbj.nig.ac.jp/general_analysis_division/ga_transfer/

また、rsyncも利用可能です。
 rsync -av "ssh -i 接続するアカウントの鍵" アカウント@gw.ddbj.nig.ac.jp:転送フォルダ ローカル環境の転送先フォルダ
 
なお、データ転送にて他に支障が発生するようであれば、連絡します。
